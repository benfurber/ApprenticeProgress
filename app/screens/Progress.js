import React, { Component } from "react";
import { Text } from "native-base";

import { Background } from "components";

class Progress extends Component {
  static navigationOptions = () => {
    return {
      title: "Progress",
    };
  }

  render() {
    return (
      <Background>
        <Text>Progress</Text>
      </Background>
    );
  }
};

export { Progress };