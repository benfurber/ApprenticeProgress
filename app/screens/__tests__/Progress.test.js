import React from "react";
import { shallow } from "enzyme";

import { Progress } from "../Progress";

describe("Progress", () => {
  it("has a valid snapshot", () => {
    const screen = shallow(<Progress />);

    expect(screen).toMatchSnapshot();
  });
});
