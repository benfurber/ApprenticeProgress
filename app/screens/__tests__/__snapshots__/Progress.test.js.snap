// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress has a valid snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Progress />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Styled(Text)>
        Progress
      </Styled(Text)>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Progress",
        "virtual": undefined,
      },
      "ref": null,
      "rendered": "Progress",
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Styled(Text)>
          Progress
        </Styled(Text)>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Progress",
          "virtual": undefined,
        },
        "ref": null,
        "rendered": "Progress",
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
